src/category_theory/limits/over.lean

```diff
@@ -182,31 +182,22 @@ instance over_has_prods_of_pullback [has_pullbacks.{v} C] (B : C) :
   has_binary_products.{v} (over B) :=
 {has_limits_of_shape := {has_limit := λ F, over_product_of_pullbacks B F}}
 
--- A collection of simp lemmas to decompose products in the over category
-@[simp]
+-- A collection of lemmas to decompose products in the over category
```

Comment:
```suggestion
/-! A collection of lemmas to decompose products in the over category -/
```
Making this a module docstring will allow it to be rendered in the mathlib docs.